nixos/doc/manual/release-notes/rl-2205.section.md

```diff
@@ -59,6 +61,21 @@ In addition to numerous new and upgraded packages, this release has the followin
 
 ## Backward Incompatibilities {#sec-release-22.05-incompatibilities}
 
+- Since systemd version 250 wireguard tunnels that are managed using networkd
+  will install routes for each of the `AllowedIPs` entries. This is in contrast
+  to the previous behaviour where routes had to be added manually.
+  To disable the new behaviour you must set the `RoutingTable=off` option in
+  the `WireGuard` section within each of your netdev definitions.
+  For example like this:
+  ```nix
+  {
+    systemd.network.netdev.my-wireguard-interface = {
+      wireguardConfig.RouteTable = "off";
+    };
+  }
```

Comment:
This defaults flip been reverted in v250.3 (which this PR doesn't include yet).

@jonringer We should bump and revert this release note section.

Reply:
it does contain 250.3

I squashed all of the bumps together

Reply:
I will revert the release notes

Reply:
reverted